tasks/chkrootkit.yml

```diff
@@ -3,5 +3,16 @@
     src: chkrootkit.conf
     dest: /etc/chkrootkit.conf
 
+- name: Add disabled chkrootkit tests to config file
+  lineinfile:
+    dest: /etc/chkrootkit.conf
+    regexp: "^DISABLED_TESTS="
+    line: "DISABLED_TESTS=\"{{CHKROOTKIT_DISABLED_TESTS | join(' ')}}\""
```

Comment:
You should make `chkrootkit.conf` a template instead of first copying it verbatim and later editing it using `lineinfile`.